### PR TITLE
feat: go lint issues and metadata schema updates

### PR DESCRIPTION
**Pull request by g-awmalik**
- addresses go linter issues across the CLI
- adds a field to designate a blueprint as deployable single or multiple times in a project
- changes the min/max fields from int to float

### CODE REVIEW

File: cli/bpmetadata/markdown_test.go [hunk 1]
```diff
@@ -212,7 +212,7 @@ func TestProcessArchitectureContent(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			content, err := os.ReadFile(path.Join(mdTestdataPath, tt.fileName))
```
Comment (apeabody) [inline cli/bpmetadata/markdown_test.go:215]: nit: Might want to also check the `os.ReadFile`s for errors.  If could provide a more useful error message than depending on the subsequent `getArchitctureInfo`.

File: cli/bpmetadata/types_ui.go [hunk 2]
```diff
@@ -58,10 +58,10 @@ type DisplayVariable struct {
 	MaximumLength int `json:"maxLength,omitempty" yaml:"maxLength,omitempty"`
 
 	// Minimum value for numeric types.
-	Minimum int `json:"min,omitempty" yaml:"min,omitempty"`
+	Minimum float32 `json:"min,omitempty" yaml:"min,omitempty"`
```
Comment (apeabody) [inline cli/bpmetadata/types_ui.go:61]: I'm assuming the change to `min` and `max` are to resolve an existing type match issue.

**Review by apeabody — APPROVED** [review]

File: cli/bpmetadata/markdown_test.go (hunk 1 above)
Comment (g-awmalik) [reply cli/bpmetadata/markdown_test.go:215]: ack

File: cli/bpmetadata/types_ui.go (hunk 2 above)
Comment (g-awmalik) [reply cli/bpmetadata/types_ui.go:61]: Yeah although, the type isn't being used yet...

**Pull request merged**